tests/common/snappi_tests/common_helpers.py

```diff
@@ -741,12 +743,13 @@ def disable_packet_aging(duthost, asic_value=None):
     Returns:
         N/A
     """
+    asic_name = duthost.get_asic_name()
     if isMellanoxDevice(duthost):
         duthost.copy(src="qos/files/mellanox/packets_aging.py", dest="/tmp")
         duthost.command("docker cp /tmp/packets_aging.py syncd:/")
         duthost.command("docker exec syncd python /packets_aging.py disable")
         duthost.command("docker exec syncd rm -rf /packets_aging.py")
-    elif isBroadcomDevice(duthost):
+    elif isBroadcomDevice(duthost) and asic_name not in invalid_pkt_aging_asic_list:
```

Comment:
config this only on dnx platforms. you can use the following check
  'platform_asic' in duthost.facts and duthost.facts['platform_asic'] == 'broadcom-dnx':

Reply:
done